scripts/BLmotors

```diff
@@ -0,0 +1,311 @@
+            lin['rport'] = -1
+            lin['rdir'] = lin['dir']
+            lin['newstyle'] = False
+        else:
+            lin['rid'] = lin['id']
+            lin['rdir'] = lin['dir']
+            lin['rhost'] = lin['host']
+            lin['rport'] = lin['port']
+            lin['newstyle'] = False
+    return config
+
+
+def findId(config):
+    """
+    This function takes the list of lists 'procmgr_config'
+    from the tuple 'config' and searchs for the IMS motors.
+    Then stores the id of them into a list.
+    """
+    configFile = config['procmgr_config']
+    listId = []
+    ims = "/ims/"
+    for confile in range(len(configFile)):
+        if (ims in configFile[confile]['dir']) and\
+           (configFile[confile]['disable'] is False):
+            listId.append(configFile[confile]['id'])
+    return listId
+
+
+def assignFullpath(listId):
+    """
+    This function creates a list with the pathname
+    for each ims motor's id from the list 'listId'
+    """
+    listsPath = []
+    for id in range(len(listId)):
+        listsPath.append('/reg/d/iocData/' + listId[id] +
+                         '/iocInfo/IOC.pvlist')
+    return listsPath
+
+
+def createListMotors(listsPath):
+    """
+    This function iterates through the listsPath, opening
+    each IOC.pvlist file to collect all the records (aliases)
+    with the IMS tag. Then it will store them in a list.
+    """
+    motors = []
+    motorsName = []
+    for path in range(len(listsPath)):
+        filename = listsPath[path]
+        with open(filename, 'r') as file:
+            for line in file:
+                comma = line.find(',')
+                if "ims" in line:
+                    motors.append(line[0:comma])
+                    motorsName.append(line[0:comma] + '.NAME')
+    return motors, motorsName
+
+
+def motorsInfo(motors, motorsName, cfg):
+    """
+    This function finds the real Pv name for each
+    record (alias), the cconfiguration name, and if
+    that configuration does matches or ot with the
+    actual configuration. Then it will create a list
+    per each record and finally it will be addes to
+    another list.
+    """
+    pm = pmgrAPI.pmgrAPI("ims_motor", cfg)
+    totalinfo = []
+    for motor in range(len(motors)):
+        infoperRecord = []
+        try:
+            motorPtwo = ophyd.signal.EpicsSignal(motorsName[motor]).get()
+            try:
+                mconf = pm.get_config(motors[motor])
+                exConfig = pm.diff_config(motors[motor])
+                if exConfig:
+                    infoperRecord.append(motors[motor])
+                    infoperRecord.append(motorPtwo)
+                    infoperRecord.append(mconf)
+                    infoperRecord.append("No")
+                else:
+                    infoperRecord.append(motors[motor])
+                    infoperRecord.append(motorPtwo)
+                    infoperRecord.append(mconf)
+                    infoperRecord.append("Yes")
```

Comment:
There's a bit of duplicated code here, two cases where we do the same thing on each side.
```suggestion
                infoperRecord.append(motors[motor])
                infoperRecord.append(motorPtwo)
                infoperRecord.append(mconf)
                if exConfig:
                    infoperRecord.append("No")
                else:
                    infoperRecord.append("Yes")
```